tests/system/small/test_session.py

```diff
@@ -20,6 +20,8 @@
 import typing
 from typing import List
 
+import google
+from google.api_core.exceptions import InternalServerError
```

Comment:
import module not class go/pystyle#imports

Reply:
Done